docs/tutorials/temporal_convolutional_network.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>I think a better header would be: "Compile the model"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/692/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
got it, thanks @Squadrick !

Reply:
updated!